tests/handlers/test_oidc.py

```diff
@@ -591,6 +598,65 @@ def test_exchange_code(self):
             yield defer.ensureDeferred(self.handler._exchange_code(code))
         self.assertEqual(exc.exception.error, "some_error")
 
+    @defer.inlineCallbacks
+    def test_extra_attributes(self):
```

Comment:
The top of this significantly copies from `test_callback`, although I couldn't really figure out a great way to abstract it. Any ideas would be appreciated.

Reply:
I unfortunately couldn't come up with a great way of making this a utility method.